src/recipe/recipe.service.ts

```diff
@@ -69,17 +76,18 @@ export class RecipeService {
     if (user.id != recipe.authorId && !recipe.isPublic) {
       throw new ForbiddenException();
     }
+    const { imageKeys, ...recipeWithoutImageKeys } = recipe;
```

Comment:
please change it, you don't need to get imageKeys

Reply:
How would You approach this then? Because i want to extract everything but `imageKeys`

Reply:
you can use lodash to omit `imageKeys` but if you don't use this library in your project then I don't need to add it for this one case and it's better to leave it as it is

const recipeWithoutImageKeys = lodash.omit(recipe, ['imageKeys']);